internal/verifier/attestation.go

```diff
@@ -96,7 +96,7 @@ func CreateAttestations(ctx context.Context, input rebuild.Input, finalStrategy
 					InvocationID: id,
 				},
 				Byproducts: []slsa1.ResourceDescriptor{
-					{Name: publicNormalizedURI, Digest: makeDigestSet(up.CanonicalHash...)},
+					{Name: publicNormalizedURI, Digest: makeDigestSet(up.StabilizedHash...)},
```

Comment:
Are we keeping normalize in the public attestation?

Reply:
For now, I'd vote to keep it to reduce churn. We hope to change the full attestation in the near to incorporate the new 'stabilize' binary better so we should revisit it again then.